test/serializable_resource_test.rb

```diff
@@ -71,6 +71,13 @@ def test_serializable_resource_with_collection_containing_errors
         }
         assert_equal serializable_resource.as_json(options), expected_response_document
       end
+
+      def test_serializable_resource_when_no_serializer_found
+        resource = ModelWithErrors.new
+        serializable_resource = ActiveModel::SerializableResource.new(resource)
+        # TODO define error to raise here
+        assert_raises { serializable_resource.as_json }
```

Comment:
@bf4, is this soemthing you want to make a PR for?

I 'think', I like this, but I'd like to see an actual implementation to see how it behaves.

esp with the case of no serializer defined, and under what real-life condition would throw that NoSerializerError error